src/promptflow-devkit/promptflow/_cli/_pf/_flow.py

```diff
@@ -199,6 +199,9 @@ def add_parser_serve_flow(subparsers):
     add_param_skip_browser = lambda parser: parser.add_argument(  # noqa: E731
         "--skip-open-browser", action="store_true", default=False, help="Skip open browser for flow serving."
     )
+    add_param_engine = lambda parser: parser.add_argument(  # noqa: E731
```

Comment:
Add promptflow-devkit/CHANGELOG.md, promptflow/CHANGELOG.md, and docs/reference/pf-command-reference.md for this.

Reply:
added